streams/how-streams-transform-data.docs.md

```diff
@@ -0,0 +1,42 @@
+# How streams transform data
+
+An attempt to use transform streams.
+
+## Requirements
+
+Input goes from 5 to 1 (both included).
+
+```js
+// Input sequence:
+// 5
+// 4
+// 3
+//
+// Output sequence:
+// { transformedChunk: 1, data: 5 }
+// { transformedChunk: 2, data: 4 }
+// { transformedChunk: 3, data: 3 }
+```
+
+## Thoughts
+
+This transformation is probably incorrect. If received numbers
+were from a file, the whole output would be mutated.
+
+Input file content:
+
+```txt
+12
+```
+
+Output file content:
+
+```txt
+{"transformedChunk":1,"data":"5"}{"transformedChunk":2,"data":"4"}
+```
```

Comment:
change comment, the example doesn't use files

Reply:
* remove marked content
* remove `more thoughts` heading because there is `thoughts` heading